application/src/main/java/uk/nhs/hee/tis/revalidation/service/RecommendationServiceImpl.java

```diff
@@ -283,26 +284,43 @@ public boolean submitRecommendation(String recommendationId, String gmcNumber,
   public TraineeRecommendationRecordDto getLatestRecommendation(String gmcId) {
     log.info("Fetching latest recommendation info for GmcId: {}", gmcId);
     final var optionalDoctorsForDB = doctorsForDBRepository.findById(gmcId);
-    final var optionalRecommendation = recommendationRepository
-        .findFirstByGmcNumberOrderByActualSubmissionDateDesc(gmcId);
-    if (!optionalDoctorsForDB.isPresent()) {
-      throw new RecommendationException(format(DOCTOR_NOT_FOUND_MESSAGE, gmcId));
-    }
+    final var recommendations = recommendationRepository.findByGmcNumber(gmcId);
+    final var draftRecommendations = recommendations.stream().filter(
+        recommendation -> recommendation.getRecommendationStatus().name().equals("READY_TO_REVIEW"))
+        .collect(Collectors.toList());
+    //Check draft recommendation: if yes return draft else normal original flow
+    if (draftRecommendations.size() > 0) {
+      final var draftRecommendation = draftRecommendations.get(0);
+      //Edge case: check if the Recommendation is submitted by other means, i.e., GMC Connect
+      //if GMC submission date is past and ignore it, i.e. return empty TraineeRecommendationRecordDto
+      if (draftRecommendation.getGmcSubmissionDate().isBefore(LocalDate.now())) {
+        return new TraineeRecommendationRecordDto();
+      }
+      draftRecommendation.setRecommendationStatus(RecommendationStatus.DRAFT);
```

Comment:
Why do we set the recommendation status to DRAFT?
In RecommendationStatus enumeration, we have DRAFT, but do we really use it?